### PR TITLE
GI02HYPM deploy csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updates per metadata review
Also updated CUIDs and removed unnecessary notes
Note - refer to attached spreadsheet for deployment depths 
*Deploy depth of ZPLSCs = top sphere design depths in column H
**Deploy depth of CTDs and WFPs = design depths in columns Q and R
***Decision was made between SW/RT that deployment depths for SIOCON = WFP depth + 10

[Subsurface_Mooring_Depth_Details_2020-05-28.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/4697944/Subsurface_Mooring_Depth_Details_2020-05-28.xlsx)

### CODE REVIEW

File: deployment/GI02HYPM_Deploy.csv [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
+AR30-03,,AR35-05,,GI02HYPM-WFP02-04-CTDPFL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-CTDPFL-00131,59.96974,-39.49001,,2560,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-05-VEL3DL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-VEL3DL-01109,59.96974,-39.49001,,2560,2669,
+AR30-03,,AR35-05,,GI02HYPM-MPM01-02-ZPLSGA009,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,,CGINS-ZPLSGA-55109,59.96974,-39.49001,,150,2669,upward-looking
+AR30-03,,AR35-05,,GI02HYPM-MPM01-02-ZPLSGA010,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,,CGINS-ZPLSGA-55111,59.96974,-39.49001,,150,2669,downward-looking
 AR35-05,,,,GI02HYPM-RIM01-00-SIOENG000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGCON-SIOCON-12368,59.97287,-39.53082,,2570,2664,
 AR35-05,,,,GI02HYPM-RIM01-02-CTDMOG039,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGINS-CTDMOG-11611,59.97287,-39.53082,,160,2664,
 AR35-05,,,,GI02HYPM-WFP02-00-WFPENG000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGCON-GWFPGX-77401,59.97287,-39.53082,,2560,2664,
```
Comment (dbogorff) [inline deployment/GI02HYPM_Deploy.csv:49]: Whew.  Looks good.  Thank you Rebecca!

**Comment by rtravis-whoi** [comment]
[Critical Metadata Revew [DT-15]](https://trello.com/c/OuquTzsl/15-critical-metadata-revew-dt-15)

File: deployment/GI02HYPM_Deploy.csv [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
+AR7-01,,AR21,,GI02HYPM-WFP02-01-FLORDL000,3,1,2016-07-11T14:29:00,2017-08-10T10:18:00,CGMGI-02HYPM-00003,CGVEH-GWFPGX-10401,CGINS-FLORDL-03280,59.9695,-39.4886,,2573,2673,
+AR7-01,,AR21,,GI02HYPM-WFP02-03-DOSTAL000,3,1,2016-07-11T14:29:00,2017-08-10T10:18:00,CGMGI-02HYPM-00003,CGVEH-GWFPGX-10401,CGINS-DOSTAL-01476,59.9695,-39.4886,,2573,2673,
+AR7-01,,AR21,,GI02HYPM-WFP02-04-CTDPFL000,3,1,2016-07-11T14:29:00,2017-08-10T10:18:00,CGMGI-02HYPM-00003,CGVEH-GWFPGX-10401,CGINS-CTDPFL-00131,59.9695,-39.4886,,2573,2673,
+AR7-01,,AR21,,GI02HYPM-WFP02-05-VEL3DL000,3,1,2016-07-11T14:29:00,2017-08-10T10:18:00,CGMGI-02HYPM-00003,CGVEH-GWFPGX-10401,CGINS-VEL3DL-01109,59.9695,-39.4886,,2573,2673,
+AR7-01,,AR21,,GI02HYPM-MPM01-02-ZPLSGA009,3,1,2016-07-11T14:29:00,2017-08-10T10:18:00,CGMGI-02HYPM-00003,,CGINS-ZPLSGA-55094,59.9695,-39.4886,,131,2673,upward-looking
+AR7-01,,AR21,,GI02HYPM-MPM01-02-ZPLSGA010,3,1,2016-07-11T14:29:00,2017-08-10T10:18:00,CGMGI-02HYPM-00003,,CGINS-ZPLSGA-55095,59.9695,-39.4886,,131,2673,downward-looking
```
Comment (snwhite) [inline deployment/GI02HYPM_Deploy.csv:28]: Deployment 3 looks good.

File: deployment/GI02HYPM_Deploy.csv [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
+AR21,,AR30-03,,GI02HYPM-WFP02-01-FLORDL000,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,CGVEH-GWFPGX-77401,CGINS-FLORDL-02350,59.9733,-39.9256,,2560,2659,
+AR21,,AR30-03,,GI02HYPM-WFP02-03-DOSTAL000,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,CGVEH-GWFPGX-77401,CGINS-DOSTAL-01106,59.9733,-39.9256,,2560,2659,
+AR21,,AR30-03,,GI02HYPM-WFP02-04-CTDPFL000,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,CGVEH-GWFPGX-77401,CGINS-CTDPFL-00097,59.9733,-39.9256,,2560,2659,
+AR21,,AR30-03,,GI02HYPM-WFP02-05-VEL3DL000,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,CGVEH-GWFPGX-77401,CGINS-VEL3DL-01025,59.9733,-39.9256,,2560,2659,
+AR21,,AR30-03,,GI02HYPM-MPM01-02-ZPLSGA009,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,,CGINS-ZPLSGA-55107,59.9733,-39.9256,,150,2659,upward-looking
+AR21,,AR30-03,,GI02HYPM-MPM01-02-ZPLSGA010,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,,CGINS-ZPLSGA-55108,59.9733,-39.9256,,150,2659,downward-looking
```
Comment (snwhite) [inline deployment/GI02HYPM_Deploy.csv:37]: Deployment 4 looks good.

File: deployment/GI02HYPM_Deploy.csv [hunk 4]
```diff
@@ -1,49 +1,49 @@
...
+AR30-03,,AR35-05,,GI02HYPM-WFP02-01-FLORDL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-FLORDL-03280,59.96974,-39.49001,,2560,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-03-DOSTAL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-DOSTAL-01476,59.96974,-39.49001,,2560,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-04-CTDPFL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-CTDPFL-00131,59.96974,-39.49001,,2560,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-05-VEL3DL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-VEL3DL-01109,59.96974,-39.49001,,2560,2669,
+AR30-03,,AR35-05,,GI02HYPM-MPM01-02-ZPLSGA009,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,,CGINS-ZPLSGA-55109,59.96974,-39.49001,,150,2669,upward-looking
+AR30-03,,AR35-05,,GI02HYPM-MPM01-02-ZPLSGA010,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,,CGINS-ZPLSGA-55111,59.96974,-39.49001,,150,2669,downward-looking
```
Comment (snwhite) [inline deployment/GI02HYPM_Deploy.csv:46]: Deployment 5 looks good.

**Pull request merged**